content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-cli.md

```diff
@@ -0,0 +1,199 @@
+
+```sh
+chainctl policies view
+```
+
+A menu will appear, prompting you to select the policy you'd like to inspect. To avoid using the interactive menu, you can follow the `view` subcommand with the name or identification number of the policy you want to inspect.
+
+```sh
+chainctl policies view enforce-policy
+```
+
+This will output the contents of the chosen policy directly.
+
+
+## Edit policies
+
+`chainctl` includes the `policies edit` subcommand, allowing you to modify existing policies.
+
+```sh
+chainctl policies edit
+```
+
+As with `policies view`, when you run `policies edit` without any arguments it will open an interactive menu from which you can select the policy that you want to edit. You can also append the name or the identification number Chainguard Enforce uses to refer to the policy to avoid having to select the policy, as in this example.
+
+```sh
+chainctl policies edit enforce-policy
+```
+
+Regardless of how you run this command, it will open your default text editor where you can edit the specified policy's contents. After saving your changes and closing the editor, `chainctl` will prompt you to confirm whether you want to save the changes to the active policy. If you do confirm, the updated policy will be saved and applied automatically.
+
+Be aware that if you used a `.yaml` file as the basis for the policy you applied to your cluster, the `edit` subcommand will not edit the original file. It will only edit the policy as it exists within Chainguard Enforce.
+
+`chainctl` also includes the `update` subcommand. Rather than updating the policy itself, this allows you to update the description of a policy.
+
+The `policies update` subcommand requires a few arguments in order to run. Specifically, you must include the name or identification number of the policy whose description you want to update, the `--description` option, and the policy's new description. This example updates the description of `enforce-policy` policy's description to read "A description of my policy."
+
+```sh
+chainctl policies update enforce-policy --description "A description of my policy."
+```
+
+You can remove a policy's description by updating it to an empty string.
+
+```sh
+chainctl policies update enforce-policy --description ""
+```
+
+
+## Delete policies
+
+If you would like to delete a policy, run the `policies delete` subcommand.
+
+```sh
+chainctl policies delete
+```
+
+As with the commands outlined previously, if you don't include any other arguments this command will open an interactive menu where you can select the policy you want to delete. It will then prompt you to confirm whether you want to delete the selected policy by pressing `Y` and then `ENTER`.
+
+To run this command non-interactively, you can append the name or identification number of the policy you want to delete followed by the `-y` flag. This will automatically answer "yes" to the confirmation prompt.
+
+```sh
+chainctl policies delete enforce-policy -y
+```
+
```

Comment:
```suggestion
You can follow this command with the `-y` flag to automatically answer "yes" to the confirmation prompt.
```

Reply:
This is just to add the closure and I think I would include it without the `-y` flag the first time so the user doesn't go automatically deleting things